telemetry/otel-collector-config.yaml

```diff
@@ -53,14 +53,7 @@ processors:
     timeout: 2s
     send_batch_max_size: 32768
 
-# Tasks that do not involve processing telemetry data. Configuring an extension does not enable it.
-extensions:
-  memory_ballast:
-    # https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/ballastextension/README.md
-    size_in_percentage: 10
```

Comment:
[The opentelemetry removal PR](https://github.com/open-telemetry/opentelemetry-collector/pull/8803/files#diff-c7c8156618a7f8126b25ca1bdfde3e172a0d2cb75c533d63a71617ae2a5c54aeR87) suggests using `GOMEMLIMIT` env var instead. Do we know what a sane value is there?

Reply:
OpenFGA is not a memory-intensive service. 1 GB of heap alloc should be fine I think.

This could change in the future, e.g. https://github.com/openfga/openfga/pull/1924 (@jpadilla)